zap/src/parser/convert.rs

```diff
@@ -682,12 +675,10 @@ impl<'src> Converter<'src> {
 								name,
 							});
 
-							return Ty::Unknown;
+							return Ty::Ref(name, Box::new(Ty::Opt(Box::new(Ty::Unknown))));
```

Comment:
I don't disagree with this change - just curious to know why you did this?

Reply:
Leftovers from a merge conflict, but I think it's better since it is more specific.